src/main/java/eladkay/scanner/init/ScannerCreativeTabs.java

```diff
@@ -0,0 +1,18 @@
+package eladkay.scanner.init;
+
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.init.Items;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+
+import static eladkay.scanner.init.ModBlocks.terrainScanner;
+
+public class ScannerCreativeTabs extends CreativeTabs {
+    public ScannerCreativeTabs(String id) {
+        super(id);
+    }
+    @Override
+    public ItemStack getTabIconItem() {
+        return new ItemStack(Item.getItemFromBlock(terrainScanner));
+    }
+}
```

Comment:
*some people* are annoyed by not having a blank line at the end of files. I'm not one of these people.
Just keep that in mind for the future but this is definitely not critical